quadratic/quadratic.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("The program finds solutions of a quadratic equation.")
+	fmt.Println("Enter the coefficients A, B and C:")
+	var A, B, C float64
+	fmt.Scan(&A, &B, &C)
+	if A == 0 {
```

Comment:
optional: it's possible to reduce code nesting by using if-elseif-else and early returns